### PR TITLE
Logging emails with generated QR code

### PR DESCRIPTION
**Pull request by nkzarrabi**
Just a prototype for what email invitations would look like. I think this is just a starting point for a discussion on what the constraints should be, pros and cons of personal invitation features..

To gain clarity on how personal invitations should be created that delete after , say, one week and delete when accepted..

Convoluted I know but again, just messing around

### CODE REVIEW

**Comment by kerkeslager** [comment]
Sorry dude, but I'm definitely not going to go this direction.

1. A _little_ bit of friction around invitations is _good_. We don't want users sending out low-effort invitations to random people. We only want people connecting if they want to connect enough to put a bit of effort into connecting. Nothing is stopping users from pasting an invitation link into an email, and we might add a "share invite" feature at some point which automates that, but we want that email coming from our users' email address, not from ours. The point of the QR codes is that we'd ideally like people to give and accept invitations _in person_, i.e. we're hanging out in person and we connect using our cell phones.
2. One principle that I'd like to enforce here is that User A can't consent to sharing User B's private data. In this case, I don't think the person sending the invitation can consent to giving us the private email which belongs to the person receiving the invitation.
3. We aren't collecting emails even from users themselves.

In short, an impersonal generated email to someone who didn't consent to us having their email address is the antithesis of everything I'm trying to do here.

There's a ton of ideas for stuff to implement on the issues tab; I'd prefer you grab one of those rather than going off and implementing stuff that isn't needed. I'd also suggest tackling some of the ones marked "good first issue" before you jump into harder stuff; I know they're not as interesting at first glance, but realistically you're going to have to learn about the codebase before you can make bigger changes to it.

I'm open to feature ideas too, but the workflow I'd like to see is Discussion -> Issue -> PR, rather than just jumping to a PR (with the exception of obvious bugs, which could just be Issue -> PR).
